bec2format/__init__.py

```diff
@@ -1,3 +1,10 @@
+import sys
+
+if sys.implementation.name == "micropython":
+    from .compatibility import mock_module
+
+    mock_module("typing")
```

Comment:
Ich würde alles Micropythonspezifische in das `.compatibility` Modul verlagern und es in `.micropython_compatibility_quirks` oder so umbennennen (Dann ist es spezifischer; `micropython_compatiblilty` könnte man so verstehen, daß es sich um einen Kompatibilitätaslayer für cpython handelt, der Micropython simuliert...).

Der Aufruf wäre dann nur noch:
```python
if sys.implementation.name == "micropython":
    import .micropython_compatibility_quirks
```

In das Modul würd ich dann alle "Hässlichkeiten" wie das mocking von typing verlagern (heisst ja auch "quirks"). Man könnte dort auch per monkey_patching fehlende Funktionen nachrüsten (solange dabei dabei keine inkompatibilitäten zu Mikropython entstehen!)...

Reply:
Wie besprochen sind jetzt für die `appnotes` keine Mocks oder andere Anpassungen für Micropython mehr nötig. Das `compatibility` Modul ist zu `micropython_compatibility_quirks` umbenannt und das Mocken von `typing` dorthin verschoben.